src/terminal/parser/stateMachine.cpp

```diff
@@ -1838,12 +1839,17 @@ void StateMachine::ProcessString(const std::wstring_view string)
     size_t start = 0;
     size_t current = start;
 
+    _currentString = string;

```

Comment:
Do I need to reset these in `_EnterGround` ? Feels like I should.

Reply:
I'm not sure about this any more. Because `ResetState()` is also called inside `ProcessString`.

Reply:
I see what you were saying, but I agree this appears as a valid exception.